src/Middleware/Localization/src/RequestLocalizationOptions.cs

```diff
@@ -11,14 +11,18 @@ namespace Microsoft.AspNetCore.Builder;
 /// </summary>
 public class RequestLocalizationOptions
 {
+    private readonly bool _useUserOverride;
+
     private RequestCulture _defaultRequestCulture =
         new RequestCulture(CultureInfo.CurrentCulture, CultureInfo.CurrentUICulture);
 
     /// <summary>
     /// Creates a new <see cref="RequestLocalizationOptions"/> with default values.
+    /// <param name="useUserOverride">Whether to use user-selected culture or default system culture. Defaults to <c>true</c>.</param>
     /// </summary>
-    public RequestLocalizationOptions()
+    public RequestLocalizationOptions(bool useUserOverride = true)
```

Comment:
I think this needs to be a separate constructor overload rather than an optional parameter.

```suggestion
    public RequestLocalizationOptions() : this(useUserOverride: true) { }
    public RequestLocalizationOptions(bool useUserOverride)
```